src/components/Upload/Forms/DynamicForm.js

```diff
@@ -152,8 +152,11 @@ class DynamicForm extends Component {
 						&& ( getFieldError(fieldName) !== undefined || getFieldValue(fieldName) === undefined)) {
 					return false;
 				}
-				if (getFieldValue(fieldName).length === 0){
-					return false;
+				if (getFieldValue(fieldName)){
+					let length = JSON.stringify(getFieldValue(fieldName)).length;
```

Comment:
I think this can be removed/simplified to limit unintended bugs while improving readability